SwiftWisdom/Rx/Rx+String.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Rx+String.swift
+//  SwiftWisdom
+//
+//  Created by Maya Saxena on 6/12/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension ObservableType where E == String {
+    public func ip_limitLength(to limit: Int) -> Observable<E> {
+        return self.map { $0.ip_trimmed(toLength: limit) }
+    }
+}
+
+extension ObservableType where E == String? {
+    public func ip_limitLength(to limit: Int) -> Observable<E> {
+        return self.map { $0?.ip_trimmed(toLength: limit) }
+    }
+}
+
+extension Reactive where Base: UITextField {
+    /** Limit the length of input to a text field
+
+    Usage Example:
+
+     textField.rx.ip_limitLength(to: 5)
+    */
+    public func ip_limitLength(to limit: Int) -> Disposable {
+        return text.ip_limitLength(to: limit).bind(to: text)
+    }
```

Comment:
I'm not sure if there's a way to handle that case here since these are Rx extensions, but I can rename the function to make it more apparent that it's only limiting user input and put in a comment